docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/00_Prerequisites_And_Assumptions/Database_Configuration.md

```diff
@@ -28,3 +28,41 @@ GRANT ALTER, DROP, CREATE, UPDATE, INSERT, SELECT, DELETE, CREATE TEMPORARY TABL
 
 This user's credentials should now be entered into your project's configuration file, `config.xml`
 within the `adminUser` and `adminPassword` tags.
+
+Tips:
+
+* ConfigSetting host value should not terminate in a slash. E.g. "http://localhost" not "http://localhost/". To fix, run: (for http) UPDATE Config SET Value='http://localhost' WHERE ConfigID=(SELECT ID FROM ConfigSettings WHERE Name='host');
+* If you get an error after clicking "Submit" or "Save data" on a form, check that url Config setting is set for your host. (Previous iterations of LORIS recommended setting this to the empty string) Run: UPDATE Config SET Value='_$yourhost_' WHERE ConfigID=(SELECT ID FROM ConfigSettings WHERE Name='url');
+* If your dashboard loads but no other modules load, ensure that your apache config file (in CentOS: /var/apache2/apache2.conf) is set to AllowOverride All in the section <Directory /var/www/> to enable re-write rules (based on htdocs/.htaccess)
+* ensure your smarty/templates_c directory is writable by apache
+* run composer update to ensure your dependencies (including smarty) are up-to-date
+
+## Create Front-end users
+
+The installation script adds admin user/password, associated with a dummy email address which should be changed. Additional users and permissions are added in the User Accounts page. For clarity and uniqueness, it is recommended to use an email address as username.  
+
+* To set/reset a user password, use the script [tools/resetpassword.php](https://github.com/aces/Loris/blob/main/tools/resetpassword.php)  
+
+```SQL
+GRANT ALTER, DROP, CREATE, UPDATE, INSERT, SELECT, DELETE, CREATE TEMPORARY TABLES, LOCK TABLES  on $dbname.* to 'lorisDBadmin'@'$dbhost' IDENTIFIED BY '$newpassword' WITH GRANT OPTION ;
+```
+
+lorisadmin is the Loris administrator Unix account. Developers may wish to have their own individual accounts.
```

Comment:
Incorrect location again. why is this part of the prerequisites to LORIS install ? obviously you are not gonna create users before installing the software. The section above describes how to create an SQL user not a loris one.

This section belongs in the Getting Started section (and some parts in the developer setup)